static/app/components/stories/themeSwitcher.tsx

```diff
@@ -0,0 +1,32 @@
+import {useCallback, useEffect, useState} from 'react';
+
+import {Button} from 'sentry/components/button';
+import {t} from 'sentry/locale';
+import ConfigStore from 'sentry/stores/configStore';
+
+export default function ThemeSwitcher() {
+  const [theme, setTheme] = useState(() => ConfigStore.get('theme'));
```

Comment:
Instead of doing this you can `useLegacyStore(ConfigStore)`. It will correctly update when the theme is changed.

Reply:
nice 👍